README.md

```diff
@@ -28,7 +28,7 @@ Clone the Synchronized-BatchNorm-PyTorch repository for
 cd Face_Enhancement/models/networks/
 git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
 cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
-cd ../../../
+cd ../../
```

Comment:
I'm pretty sure that we doesn't need this change. Take a look on l28.

Reply:
After following the README.md on fresh instdall from the repo, that path was wrong.